exercises/helloworld-workflow/solution/Client/Client.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>

```

Comment:
Some of this common stuff can be put in the root in a file called `Directory.Build.Props` so you don't have to type it in each `.csproj` file.

Also, you should consider some of the stuff we have in https://github.com/temporalio/samples-dotnet/blob/main/Directory.Build.props with `EnableNETAnalyzers` and `TreatWarningsAsErrors` so you're getting strict checking of your C# code. And you can add https://github.com/temporalio/samples-dotnet/blob/main/.editorconfig or similar at the root and end your workflow files with `.workflow.cs` (because some workflow norms are not C# norms, see https://github.com/temporalio/sdk-dotnet/?tab=readme-ov-file#workflow-editorconfig).

Reply:
Bump